site/includes/sign-on-off.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+	<div class="row">
+{{#is signed "off"}}
```

Comment:
There should be no `off` mode here. The register/login interface is not in the sign-on information bar but directly in the content area.